spec/Section 3 -- Type System.md

```diff
@@ -68,7 +68,7 @@ schema {
 }
 
 """
-Root type for all your queries
+Root type for all your query operations
```

Comment:
Is this a valid use of the term `query operation`? According to the spec and the glossary I believe a query operation is an `OperationDefinition` with the type `query`.

If this is not a valid use of the term, what term should be used to describe these fields on the Query type?

Reply:
I recently had a conversation with a colleague here at Indeed and use (misused?) the term `query operation` in a similar context.

I said something like "when a response includes data for multiple query operations...". He [pointed me to the spec](http://spec.graphql.org/draft/#GetOperation()) and pointed out that only one `query operation` can be executed.

What I was trying to communicate was that it's possible to select multiple fields on the top level Query type.

Reply:
`Query` is the root type for all query operations; you issue a selection set against the Query root type, this selection set may select more than one field like most selection sets (the exception being the selection set at the root of the `subscription` operation).

Here's a (single) query operation issuing a selection set against a root Query type:

```graphql
query {
  __typename
  myField
  myOtherField { id }
}
```

So yes, I believe the usage here is correct.

>  According to the spec and the glossary I believe a query operation is an OperationDefinition with the type query.

Correct.

> He pointed me to the spec and pointed out that only one query operation can be executed.

He's correct :+1: 

> What term should be used to describe these fields on the Query type?

You could call them "Query fields" I guess; just like fields on the User type would by "User fields." I'm not sure we have an established name for them, the root level fields on the `mutation` operation would generally be called "mutation fields" and for `subscription` "subscription fields" so I think "query fields" would make sense, but is potentially ambiguous.

Reply:
FWIW the historical official name for these were "root fields" - when we generalized roots to be no different from other types, we dropped an official name. I've also heard "top level fields" which makes sense to me, but "Query fields" or "Fields on the Query type" both would be perfectly clear